src/UnityExtension/Assets/Editor/GitHub.Unity/UI/SettingsView.cs

```diff
@@ -180,21 +180,24 @@ private void MaybeUpdateData()
             {
                 if (cachedUser == null || String.IsNullOrEmpty(cachedUser.Name))
                 {
-                    var user = new User();
-                    GitClient.GetConfig("user.name", GitConfigSource.User)
-                        .Then((success, value) => user.Name = value).Then(
-                    GitClient.GetConfig("user.email", GitConfigSource.User)
-                        .Then((success, value) => user.Email = value))
-                    .FinallyInUI((success, ex) =>
-                    {
-                        if (success && !String.IsNullOrEmpty(user.Name))
+                    if(GitClient != null)
```

Comment:
This if should be with the condition above:

`if ((cachedUser == null || String.IsNullOrEmpty(cachedUser.Name)) && GitClient != null)`